client/src/components/Register/Register.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { addUser, getUsers, loginUser } from "../../helpers/api";
+
+class Register extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      email: "",
+      password: "",
+      users: [],
+      message: "",
+      messageAlert: ""
+    };
+  }
+  componentDidMount = async () => {
+    const res = await getUsers();
+    const data = res;
```

Comment:
this line is redundant. Why create unnecessary variable if they are the same. You can call `res` as `users` and then setState with just `{ users }`, it's the same as `{users: users}`.

Reply:
you are right again. I didnt see that. I did this at night. I will change that

Reply:
👍